mlir/include/mlir/Analysis/Presburger/IntegerRelation.h

```diff
@@ -545,6 +557,11 @@ class IntegerRelation {
 
   void removeDuplicateDivs();
 
+  /// Simplify iteratively attempt to remove redundant equations by Gaussian
+  /// elimination and attempt to remove duplicate inequalities until a fixed
+  /// point is reached.
```

Comment:
I think you can just write the doc as "Simplify the constraint system by removing canonicalizing constraints and removing redundant constraints".